dev/reference/tidb-binlog-overview.md

```diff
@@ -33,7 +33,7 @@ Drainer collects and merges binlogs from each Pump, converts the binlog to SQL o
 
 ### `binlogctl` guide
 
-[`binlogctl`](https://github.com/pingcap/tidb-tools/tree/master/tidb-binlog/binlogctl) is an operations tool for TiDB Binlog with the following features:
+`binlogctl` is an operations tool for TiDB Binlog with the following features:
```

Comment:
```suggestion
`binlogctl` is an operations tool for TiDB Binlog with the following features:
```
Link `binlogctl` to https://pingcap.com/docs-cn/v3.0/how-to/get-started/tidb-binlog/#binlogctl?

Reply:
OK